Core/fft_mt_r2iq.h

```diff
@@ -7,7 +7,7 @@
 #include <string.h>
 
 // use up to this many threads
-#define N_MAX_R2IQ_THREADS 4
+#define N_MAX_R2IQ_THREADS 1
```

Comment:
what is the issue when having multi threads?

Reply:
Segmentation fault happened for me somewhere in libfft if I enable multi threads (even in unittest). I will try to compile recent libfft from source and see what changed.

But anyway, r2iq not currently used in libsddc, I just forgot to revert the changes.

Reply:
I reverted this to original state